cloud_chatops/pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-pythonpath = lib
+pythonpath = src
```

Comment:
Shouldn't this be simply `.` or blank? Since we're now missing the lib folder
I would have thought the `from src import X` in `src/__init__.py` does this for us?

Reply:
Not sure I understand what you mean here. pythonpath is used to add the dirs to the python search path for imports. Hence src makes sense here as its the root module of code.